src/chatterbox/tts.py

```
@@ -163,145 +155,200 @@ def from_local(cls, ckpt_dir, device) -> 'ChatterboxTTS':
+        tokens_per_slice=None,
+        remove_milliseconds=None,
+        remove_milliseconds_start=None,
+        chunk_overlap_method=None,
+        # cache optimization params
+        max_new_tokens=750, 
+        max_cache_len=1050, # Affects the T3 speed, hence important
+        # t3 sampling params
+        repetition_penalty=1.2,
+        min_p=0.05,
+        top_p=1.0,
+        t3_params={},
```

Ruff (0.13.1)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)